### ISSUE TITLE
Creature su Scratch

### ISSUE THREAD

**Issue by sfederici** [Mittagskogel/Sulfurous#61]
Texts do not show fully.

https://scratch.mit.edu/projects/48787568/
https://sulfurous.aau.at/#48787568
Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/61.0.3163.100 Safari/537.36

**Comment by FRALEXAlexander** [comment]
Text now shows up but the blank spaces are missing.

**Comment by sfederici** [comment]
Thanks a lot. Out of curiosity: are you planning (or, maybe, you already did) to support Scratch 3 projects?

**Comment by demesos** [comment]
Yes, we plan to support .SB3 as well, will come in July this year.

**Comment by sfederici** [comment]
Great. Thanks
